x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/collapsed_item_actions_new.tsx

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
Is there a reason for the file to have a suffix of `_new`? The file name no longer matches what it exports `CollapsedItemActions`.

Reply:
I will remove this suffix and old menu 👍